build.rs

```diff
@@ -60,7 +60,5 @@ fn dcmds() -> Vec<CommandDescription> {{
 
     write!(output, "    ]\n}}")?;
 
-    println!("cargo:rerun-if-changed=Cargo.toml");
```

Comment:
Is it legit to remove this?  It doesn't look like `MetadataCommand::new().manifest_path("./Cargo.toml").exec().unwrap()` prints `rerun-if-changed`, so this line might still be needed.

In fact, we might want to move this line above the `MetadataCommand::new()` call, to ensure that `build.rs` reruns on `Cargo.toml` changes even if it errors out sometime _after_ reading `Cargo.toml` (avoiding stuff like https://github.com/oxidecomputer/hubris/pull/836)

Reply:
I'm open to that! I couldn't remember why I added it, and so I thought I should remove it. Happy to move it up instead; I do think that's the right call, and I'll add a comment so I remember why next time.